clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -128,9 +129,11 @@ final case class QueryWgsCram(@Recurse queryInput: TransferWgsCramV1QueryInput,
     extends QueryCommand(WgsCramIndex)
 
 @CommandName(ClioCommand.moveWgsCramName)
-final case class MoveWgsCram(@Recurse key: TransferWgsCramV1Key,
-                             destination: URI)
-    extends MoveCommand(WgsCramIndex)
+final case class MoveWgsCram(
+  @Recurse key: TransferWgsCramV1Key,
+  destination: URI,
+  override val newSamplePrefix: Option[String] = None
```

Comment:
You could throw [`@Hidden`](https://github.com/alexarchambault/case-app/blob/master/annotations/shared/src/main/scala/caseapp/Annotations.scala#L62) on this so it won't show up in the help message.

If we really want to wall it off from command-line users, we could set it as a new constructor parameter on the `MoveExecutor` instead of the `MoveCommand`.

Reply:
What do you think.  Probably a good idea until we want to expose it.

Reply:
I'd rather it be part of the executor.

Reply:
Not sure how to plumb it to here though.  Ideas?

https://github.com/broadinstitute/clio/blob/develop/clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/CommandDispatch.scala#L16

Reply:
Ah. Never mind.